### PR TITLE
added watchdog and empty python module

### PR DESCRIPTION
**Pull request by marc-hanheide**
to gazebosim

* now launched the twist_watchdog to stop robot turning in sim when no cmd is receive
* installs the Python script correctly
* also added empty Python package for futre extensions.

### CODE REVIEW

File: src/limo_gazebosim/launch/limo_gazebo_diff.launch.py [hunk 1]
```diff
@@ -191,7 +191,13 @@ def generate_launch_description():
                     '-z', spawn_z_val,
                     '-Y', spawn_yaw_val],
                     output='screen')
- 
+
+  twist_watchdog = Node(
+    package='uol_turtlebot_simulator',
```
Comment (gcielniak) [inline src/limo_gazebosim/launch/limo_gazebo_diff.launch.py:196]: shouldn't that be limo_gazebosim?

**Review by gcielniak — COMMENTED** [review]
just a small naming issue but probably without consequences

**Comment by marc-hanheide** [comment]
> just a small naming issue but probably without consequences

no! a really important one! Just coincidence it worked as I had that package installed...

**Comment by marc-hanheide** [comment]
could you fix and test? I'm on phone only right now

**Pull request merged**